modules/gdscript/gdscript_utility_callable.cpp

```diff
@@ -80,6 +80,20 @@ ObjectID GDScriptUtilityCallable::get_object() const {
 	return ObjectID();
 }
 
+int GDScriptUtilityCallable::get_argument_count(bool &r_is_valid) const {
+	switch (type) {
+		case TYPE_GLOBAL:
+			r_is_valid = true;
+			return Variant::get_utility_function_argument_count(function_name);
+		case TYPE_GDSCRIPT:
+			r_is_valid = true;
+			return GDScriptUtilityFunctions::get_function_argument_count(function_name);
+		default:
+			r_is_valid = false;
+			return 0;
```

Comment:
I would prefer an explicit `case TYPE_INVALID` instead of `default` for consistency. If we potentially add a new type and forget to add the new case here, the compiler will produce a warning that is treated as an error.

Reply:
Will need a default path with a warning to accomplish that unfortunately, but will add that, it complains of non-returning path with just the three cases